.github/workflows/release-api-package.yml

```diff
@@ -0,0 +1,77 @@
+name: Release api package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+          registry-url: https://npm.pkg.github.com/
+
+      - name: Log in to GitHub Package Registry
+        run: |
+          echo "//npm.pkg.github.com/:_authToken=\${{ secrets.RELEASE_TOKEN }}" > .npmrc
+          npm config set @litentry/litentry-parachain:registry https://npm.pkg.github.com
+
+      - name: Install pnpm
+        run: npm install -g pnpm
```

Comment:
Shouldn't we rather `corepack enable pnpm` here to make sure it uses the version set in the `package.json`? :thinking:

Reply:
Thanks! fixed!